backend/services/kakao_pay_service.py

```
@@ -0,0 +1,255 @@
+            prepare_data = {
+                "cid": self.cid,
+                "partner_order_id": partner_order_id,
+                "partner_user_id": partner_user_id,
+                "item_name": payment_request.product_name,
+                "quantity": payment_request.quantity,
+                "total_amount": payment_request.total_amount,
+                "vat_amount": vat_amount,
+                "tax_free_amount": payment_request.tax_free_amount,
+                # 결제 완료 후 리다이렉트 URL들
+                # TODO: 환경변수로 분리하여 개발/운영 환경별 관리 필요
+                "approval_url": f"http://localhost:3000/payment/success",  
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)